gnmpagerduty/__init__.py

```diff
@@ -0,0 +1,8 @@
+from plugin import *
+from tasks import check_storage
+
+__version__ = 'DISPLAY_VERSION'
+__production__ = False
+
+check_storage.delay()
```

Comment:
i think you have to run a manage.py command to update the tasks, maybe?  I use timed tasks in purgemeister, take a look at how it works in there for syntax and have a look at the rpm specfile for the installation script.

The issue might be that you are manually installing the code only on one node of the two-node dev cluster; possibly the other one is running celerybeat that would trigger this task.

Try cloning https://github.com/fredex42/portal3vm to get a fresh development environment and see if it runs on there.